generators/app/templates/src/layouts/mainGuide.soy

```diff
@@ -0,0 +1,15 @@
+{namespace mainGuide}
```

Comment:
I think we should try to keep the generated markup as simple as possible, and at first glance having `main`, `guide`, and `mainGuide` layouts is a little confusing, any way we can get rid of this?